docs/source/tutorial.rst

```diff
@@ -206,15 +229,15 @@ Let’s see what type of value we get for different integrators.
 
 .. parsed-literal::
 
-    **Output:** Results: 21.83991813659668
-            Abs. Error: 9.06180573e+00
-            Rel. Error: 7.09166229e-01
+    **Output:** RuntimeError: result type ComplexFloat can't be cast to the desired output type Float
     
 
-Notably, Simpson’s method is already sufficient for a perfect solution here with 101 points. 
+Notably, Simpson's and Boole's methods are already really good here with only 101 points. 
+(You can try setting the function ``f(x) = torch.exp(x) * torch.pow(x,2)`` instead 
+to see Simpson's and Boole's methods find the perfect solution).
```

Comment:
>  (You can try setting the function ``f(x) = torch.exp(x) * torch.pow(x,2)`` instead 
to see Simpson's and Boole's methods find the perfect solution).

I don't understand this? It's a harder function so the errors will be higher? So not perfect?

Reply:
No, we had 
`torch.exp(3.0j*x) * torch.pow(x,2)`
and then I suggest using
`torch.exp(x) * torch.pow(x,2)`

Reply:
Ah, so it's not harder. However, e^x is not a polynomial, so it cannot be "perfectly" solved by either method. I would just remove this as it does not really add to informing the reader.